NCPCore/src/main/java/fr/neatmonster/nocheatplus/utilities/map/BlockProperties.java

```diff
@@ -2807,25 +2807,13 @@ public static final boolean isGround(final Material id, final long ignoreFlags)
      */
     public static final boolean isAir(Material type) {
         // TODO: Flags rather?
-        return type == null || type == Material.AIR
+        return type != null && (type == Material.AIR
                 // Assume the compiler throws away further null values.
                 || type == BridgeMaterial.VOID_AIR
-                || type == BridgeMaterial.CAVE_AIR
+                || type == BridgeMaterial.CAVE_AIR)
                 ;
     }
 
-    /**
-    * Convenience method excluding null as air result.
-    *
-    * @param type
-    *            the type
-    * @return true, if is air
-    */
-    public static final boolean isActuallyAir(Material type) {
-        
-        return type != null && (type == Material.AIR || type == Material.CAVE_AIR || type == Material.VOID_AIR);
-    }
-
```

Comment:
One more thing, I meant still keep isActuallyAir() with type != null && isAir(type). Not sure change directly on isAir() caused anything when it is using anywhere!